release/_package.sh

```diff
@@ -132,6 +132,23 @@ function package_release {
 
 	lc_cd "${_BUILD_DIR}/release"
 
+	if [[ "${_PRODUCT_VERSION}" == 7.3* ]]
+	then
+		mkdir -p liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		cp "${_PROJECTS_DIR}"/liferay-portal-ee/lib/development/hsql.jar liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		cp "${_PROJECTS_DIR}"/liferay-portal-ee/lib/global/*.jar liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		cp "${_PROJECTS_DIR}"/liferay-portal-ee/portal-kernel/portal-kernel.jar liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		find "${_PROJECTS_DIR}"/liferay-portal-ee/tools/sdk/dist -name 'com.liferay.registry.api-*.jar' -exec cp {} liferay-portal-dependencies-"${_PRODUCT_VERSION}" \;
```

Comment:
@pgydev, is it covering all these instructions?

Reply:
@natocesarrego  What do you mean by that?